doc/API.md

```diff
@@ -110,11 +110,13 @@ All the endpoints have the optional parameter `limit` (10 by default) where you
 
   **Required:**
 
-  `nwlat=[northwest latitude],selat=[southeast latitude],nwlng=[northwest longitude],selng=[southeast longitude]`: search notes within the rectangle boundary made by these diagonally opposite coordinates.
+  `nwlat=[northwest latitude],selat=[southeast latitude],nwlng=[northwest longitude],selng=[southeast longitude]`: Search users within the rectangle boundary made by these diagonally opposite coordinates.
 
   **Optional:**
 
-  `tag=[string]`: the search can be refined by passing a tag field.
+  `tag=[string]`: The search can be refined by passing a user tag field.
+
+  `field=[string]`: Accepts the value `node_tag` for searching users following the node tag (topic).
```

Comment:
I'll go ahead and merge this. Maybe though we could be extra clear above that the default is user_tag and that this switches the use of the tag parameter to link node tags instead... Maybe just as a quick follow-up? Thanks!!